<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="editEntry">
      <property name="text">
       <string>Edit Cat</string>
      </property>
      <property name="icon">
       <iconset theme="document-edit"/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="deleteEntry">
      <property name="text">
       <string>Delete Car</string>
      </property>
      <property name="icon">
       <iconset theme="list-remove"/>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <layout class="QVBoxLayout" name="filters"/>
    </item>
    <item row="2" column="6">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addEntry">
      <property name="text">
       <string>Add Car</string>
      </property>
      <property name="icon">
       <iconset theme="list-add"/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="sortSelector"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="openFile">
      <property name="text">
       <string>Open File</string>
      </property>
      <property name="icon">
       <iconset theme="document-open"/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTableWidget" name="mainTable">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="newFile">
      <property name="text">
       <string>New File</string>
      </property>
      <property name="icon">
       <iconset theme="document-new"/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sort By:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
